### DISCUSSION TITLE
Beer CSS in Dotnet Blazor

### DISCUSSION THREAD

**Discussion by Kristianfriis** [beercss/beercss#176] [Show and tell]
Hi all, 

I have been working on a Blazor Component Library for a personal project. It is simply wrapping BeerCss into some reusable components with enums to provide type safety in applying classes to the components. 
It was entangled in my other project, so have pulled it out into its own repository, but I am adding the components as I need them. Thus the amount of components is not vast right now. 

Hopefully I can keep adding components, so at some point it becomes a complete Beer CSS Component library for Blazor. 
It is on github here: [link](https://github.com/Kristianfriis/BlazorBeerCss)

Any thoughts or comments are more than welcome :)

**Comment by laurentpayot** [comment]
Nice!

**Comment by Kristianfriis** [comment]
I have added a showcase as a Blazor WASM application. 
It is a little bit funky at times. For example navigation is a bit funky 😄 

Again, not all components are there, but it is a good start :) 
[DemoLink](https://kristianfriis.github.io/BlazorBeerCss/)

**Comment by leonardorafael** [comment]
Hi,

We can create a section in README.md called "Projects developed by community". But first, we need to create a review process to add the project to README.md.

There are some people that are developing different things using the framework. I don't have idea how we can review the projects yet. What will be the prerequisites, etc...
